beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -714,3 +714,61 @@ func (f *ForkChoice) ParentRoot(root [32]byte) ([32]byte, error) {
 	}
 	return n.parent.root, nil
 }
+
+// OnPayloadAttestationMessage processes a new aggregated payload attestation message.
+func (s *Store) OnPayloadAttestationMessage(
```

Comment:
I dont think we should call this function `OnPayloadAttestationMessage`. It's not implementing all the [on_payload_attestation_message](https://github.com/ethereum/consensus-specs/blob/74124f521b4c0d1bd68ac92595177ec8fcc0e931/specs/_features/eip7732/fork-choice.md#on_payload_attestation_message) so it's a bit confusing.  We should just call this `UpdatePTCVotes`

Reply:
I'm aware, I need to add more code here

Reply:
But I'm also not sure if we want to implement the entire function in fork choice, I thought we may just want to update the votes only?

Reply:
makes sense I've limited this PR to updating Votes on payloadAttestation